src/ray/raylet/scheduling/cluster_resource_scheduler.h

```diff
@@ -366,6 +372,14 @@ class ClusterResourceScheduler {
   // resources availabile at that node is 0.2 + 0.3 + 0.1 + 1. = 1.6
   void UpdateLocalAvailableResourcesFromResourceInstances();
 
+  /// Populate the relevant parts of the heartbeat table. This is intended for
+  /// sending raylet <-> gcs heartbeats. In particular, this should fill in
+  /// resources_available and resources_total.
+  void Heartbeat(std::shared_ptr<HeartbeatTableData> data,
```

Comment:
Can you document the args?

Also, I think the convention is to list output parameters after input parameters.